xonsh/platform.py

```diff
@@ -173,14 +173,14 @@ def pathbasename(p):
 @functools.lru_cache(1)
 def githash():
```

Comment:
sure, and maybe I could also change this function name to `get_git_version_info()`?